controllers/auth/signupController.js

```diff
@@ -21,7 +21,7 @@ export default class SignUpController {
    * @return {object} - It returns the response object.
    */
   static async signUp(req, res) {
-    const { email, password, username } = req.body;
+    const { email, password, username, roleId } = req.body;
```

Comment:
@niomwungeri-fabrice I think the roleId is not necessary on sign up and the user should be given the lowest role as default